src/apiServices/AuthServiceNode.ts

```diff
@@ -0,0 +1,96 @@
+import { postRequest, getRequest } from '../http'
+
+import { loginUser } from '../auth/loginUser'
+
+import { getHeaders } from '../util/getHeaders'
+import { errorHandler } from '../util/errorHandler'
+import { API_URL_PRODUCTION } from '../util/constants'
+
+import { JwtTokensInterface } from '..'
+
+export class AuthServiceNode {
+
+    public ethAddress: string
+    public password: string
+    public jwtTokens: JwtTokensInterface = {
+        access: '',
+        refresh: ''
+    }
+
+    constructor(ethAddress?: string, password?: string) {
+        this.ethAddress = ethAddress as string
+        this.password = password as string
+    }
+
+    public async login(): Promise<JwtTokensInterface | undefined> {
+        try {
+            const result = await loginUser(this.ethAddress, this.password, `${API_URL_PRODUCTION}/auth/login`)
```

Comment:
Perhaps the api url should be set in the constructor, and `${API_URL_PRODUCTION}/auth/login` should be changed to `${this.apiUrl}/auth/login`. That way it is easier to configure when we end up adding a way to do that.